src/data/ACTIONS/root/SGE.ts

```diff
@@ -1,5 +1,298 @@
+		cooldown: 60000,
+		statusesApplied: ['PHYSIS'],
+	},
+	PHYSIS_II: {
+		id: SGE_STUB + 9,
+		name: 'Physis II',
+		icon: '',
+		cooldown: 180000,
+		statusesApplied: ['PHYSIS_II'],
+	},
+	HOLOS: {
+		id: SGE_STUB + 10,
+		name: 'Holos',
+		icon: '',
+		cooldown: 60000,
+	},
+	PEPSIS: {
+		id: SGE_STUB + 11,
+		name: 'Pepsis',
+		icon: '',
+		cooldown: 15000,
+	},
+
+	/** Utility */
+	KARDIA: {
+		id: SGE_STUB + 12,
+		name: 'Kardia',
+		icon: '',
+		cooldown: 5000,
+	},
+	EUKRASIA: { // TODO: Apparently this is on the GCD? Will need confirmation after launch
+		id: SGE_STUB + 13,
+		name: 'Eukrasia',
+		icon: '',
+		onGcd: true,
+		gcdRecast: 1000,
+	},
+	SOTERIA: {
+		id: SGE_STUB + 14,
+		name: 'Kardia',
+		icon: '',
+		cooldown: 90000,
+		statusesApplied: ['SOTERIA'],
+	},
+	ICARUS: {
+		id: SGE_STUB + 15,
+		name: 'Icarus',
+		icon: '',
+		cooldown: 45000,
+	},
+	ZOE: {
+		id: SGE_STUB + 16,
+		name: 'Zoe',
+		icon: '',
+		cooldown: 90000,
+		statusesApplied: ['ZOE'],
+	},
+	KRASIS: {
+		id: SGE_STUB + 17,
+		name: 'Krasis',
+		icon: '',
+		cooldown: 60000,
+		statusesApplied: ['KRASIS'],
+	},
+
+	/** Addersgall abilities */
+	RIZOMATA: {
+		id: SGE_STUB + 18,
+		name: 'Rizomata',
+		icon: '',
+		cooldown: 90000,
+	},
+	DRUOCHOLE: {
+		id: SGE_STUB + 19,
+		name: 'Druochole',
+		icon: '',
+		cooldown: 1000,
+	},
+	IXOCHOLE: {
+		id: SGE_STUB + 20,
+		name: 'Ixochole',
+		icon: '',
+		cooldown: 30000,
+	},
+	KERACHOLE: {
+		id: SGE_STUB + 21,
+		name: 'Kerachole',
+		icon: '',
+		cooldown: 30000,
+		statusesApplied: ['KERACHOLE', 'KERACHOLE_REGEN'],
+	},
+	TAUROCHOLE: {
+		id: SGE_STUB + 22,
+		name: 'Taurochole',
+		icon: '',
+		cooldown: 30000,
+		statusesApplied: ['TAUROCHOLE'],
+	},
+
+	/** Single-Target DPS Spells */
+	DOSIS: {
+		id: SGE_STUB + 23,
+		name: 'Dosis',
+		icon: '',
+		onGcd: true,
+		speedAttribute: Attribute.SPELL_SPEED,
+		castTime: 1500,
+		mpCost: 400,
+	},
+	DOSIS_II: {
+		id: SGE_STUB + 24,
+		name: 'Dosis II',
+		icon: '',
+		onGcd: true,
+		speedAttribute: Attribute.SPELL_SPEED,
+		castTime: 1500,
+		mpCost: 400,
+	},
+	DOSIS_III: {
+		id: SGE_STUB + 25,
+		name: 'Dosis III',
+		icon: '',
+		onGcd: true,
+		speedAttribute: Attribute.SPELL_SPEED,
+		castTime: 1500,
+		mpCost: 400,
+	},
+	EUKRASIAN_DOSIS: {
+		id: SGE_STUB + 26,
+		name: 'Eukrasian Dosis',
+		icon: '',
+		onGcd: true,
+		gcdRecast: 1500,
+		mpCost: 400,
+		statusesApplied: ['EUKRASIAN_DOSIS'],
+	},
+	EUKRASIAN_DOSIS_II: {
+		id: SGE_STUB + 27,
+		name: 'Eukrasian Dosis II',
+		icon: '',
+		onGcd: true,
+		gcdRecast: 1500,
+		mpCost: 500,
+		statusesApplied: ['EUKRASIAN_DOSIS_II'],
+	},
+	EUKRASIAN_DOSIS_III: {
+		id: SGE_STUB + 28,
+		name: 'Eukrasian Dosis III',
+		icon: '',
+		onGcd: true,
+		gcdRecast: 1500,
+		mpCost: 600,
+		statusesApplied: ['EUKRASIAN_DOSIS_III'],
+	},
+
+	/** AoE DPS Spells */
+	TOXIKON: {
+		id: SGE_STUB + 29,
+		name: 'Toxikon',
+		icon: '',
+		onGcd: true,
+		speedAttribute: Attribute.SPELL_SPEED,
+		gcdRecast: 2500,
```

Comment:
I assume it doesn't hurt anything but do we need to specify a 2.5s GCD recast on this and the three spells below it?

Reply:
This was just my brain melting by the time I got here. I think it was meant for the Phlegma spells, since those are on the GCD, affected by spell speed, but also have a cooldown _and_ charges  (:blobsweat:). 'Tis fixed now.